telemetry-http-java11/src/main/java/com/newrelic/telemetry/Java11Http.java

```diff
@@ -0,0 +1,29 @@
+package com.newrelic.telemetry;
+
+import com.newrelic.telemetry.http.HttpPoster;
+import java.util.function.Supplier;
+
+/** Simple way to create a TelemetryClient that is backed by a Java11+ based HTTP poster. */
+public class Java11Http {
+
+  private static final Supplier<HttpPoster> supplier = Java11HttpPoster::new;
```

Comment:
It feels a little odd to have this up here in a final var. Fewer characters in the file just to use it at the call site, I think.

Reply:
ok, that's some reasonable duplication.